File: Repository/Repositories/Attendance/EngagementFlags/FlaggedStudentReportItem.cs
namespace Repository.Repositories.Attendance.EngagementFlags;

public class FlaggedStudentReportItem
{
    public string StudentName { get; set; }
    public string StudentEmail { get; set; }
    public string GradeLevel { get; set; }
    public string MentorName { get; set; }
    public string WeekOfDate { get; set; }
    public InterventionLevel Level { get; set; }
    public InterventionStatus Status { get; set; }
}

File: Web/Angular/ClientApp/src/app/models/flagged-student-report-item.ts
import { InterventionLevel } from "../enums/intervention-level.enum";
import { InterventionStatus } from "../enums/intervention-status.enum";

export interface FlaggedStudentReportItem {
  studentName: string;
  studentEmail: string;
  gradeLevel: string;
  weekOfDate: string;
  mentorName: string;
  level: InterventionLevel;
  status: InterventionStatus;
}
